cli/display.rs

```diff
@@ -3,11 +3,10 @@
 /// A function that converts a float to a string the represents a human
 /// readable version of that number.
 pub fn human_size(size: f64) -> String {
-  let negative = if size.is_sign_positive() { "" } else { "-" };
   let size = size.abs();
   let units = ["B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
   if size < 1_f64 {
-    return format!("{}{}{}", negative, size, "B");
+    return format!("-{}{}", size, "B");
```

Comment:
I think this would add a negative sign for `human_size(0_f64)`?

Reply:
Oh sorry, my understanding was wrong 😅
this change is not correct.
But test cases seems like not enough (e.g -10GB)
I’ll revert and add test cases in this PR.

Reply:
Done revert and add tests :)